src/Bigtable/Connection/TableAdminConnectionInterface.php

```diff
@@ -0,0 +1,254 @@
+     *          * Key assignment:
+     *              - Tablet 1 `[, apple)                => {"a"}.`
+     *              - Tablet 2 `[apple, customer_1)      => {"apple", "custom"}.`
+     *              - Tablet 3 `[customer_1, customer_2) => {"customer_1"}.`
+     *              - Tablet 4 `[customer_2, other)      => {"customer_2"}.`
+     *              - Tablet 5 `[other, )                => {"other", "zz"}.`
+     *     @type \Google\GAX\RetrySettings|array $retrySettings
+     *          Retry settings to use for this call. Can be a
+     *          {@see Google\GAX\RetrySettings} object, or an associative array
+     *          of retry settings parameters. See the documentation on
+     *          {@see Google\GAX\RetrySettings} for example usage.
+     * }
+     * @return TBD
+     *
+     */
+    public function createTable($tableId, array $optionalArgs = []);
+
+    /**
+     * Creates a new table in the specified instance with column family.
+     * The table can be created with a full set of initial column families,
+     * specified in the request.
+     *
+     * @param string $tableId      The name by which the new table should be referred to within the parent
+     *                             instance, e.g., `foobar` rather than `<parent>/tables/foobar`.
+     * @param string $columnFamily e.g., `cf`
+     *
+     * @param array  $optionalArgs {
+     *                             Optional.
+     *
+     *     @type Split[] $initialSplits
+     *          The optional list of row keys that will be used to initially split the
+     *          table into several tablets (tablets are similar to HBase regions).
+     *          Given two split keys, `s1` and `s2`, three tablets will be created,
+     *          spanning the key ranges: `[, s1), [s1, s2), [s2, )`.
+     *
+     *          Example:
+     *
+     *          * Row keys := `["a", "apple", "custom", "customer_1", "customer_2",`
+     *                         `"other", "zz"]`
+     *          * initial_split_keys := `["apple", "customer_1", "customer_2", "other"]`
+     *          * Key assignment:
+     *              - Tablet 1 `[, apple)                => {"a"}.`
+     *              - Tablet 2 `[apple, customer_1)      => {"apple", "custom"}.`
+     *              - Tablet 3 `[customer_1, customer_2) => {"customer_1"}.`
+     *              - Tablet 4 `[customer_2, other)      => {"customer_2"}.`
+     *              - Tablet 5 `[other, )                => {"other", "zz"}.`
+     *     @type \Google\GAX\RetrySettings|array $retrySettings
+     *          Retry settings to use for this call. Can be a
+     *          {@see Google\GAX\RetrySettings} object, or an associative array
+     *          of retry settings parameters. See the documentation on
+     *          {@see Google\GAX\RetrySettings} for example usage.
+     * }
+     * @return TBD
+     *
+     */
+    public function createTableWithColumnFamily($tableId, $columnFamily, array $optionalArgs = []);
```

Comment:
`createTableWithColumnFamily` should not be needed.  `createTable` should be able to handle this.